flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisConfigOptions.java

```diff
@@ -291,4 +292,26 @@ public static Properties getStreamLoadProp(Map<String, String> tableOptions) {
         }
         return streamLoadProp;
     }
+
+    public static LinkedHashMap<String,String> getJdbcUrlDefaultParameter() {
+        final LinkedHashMap defaultParameter = new LinkedHashMap<String,String>();
+        defaultParameter.put("rewriteBatchedStatements", "true");
+        return defaultParameter;
```

Comment:
Why do we need to add this parameter?

Reply:
https://dev.mysql.com/doc/connectors/en/connector-j-connp-props-performance-extensions.html 
https://vladmihalcea.com/mysql-rewritebatchedstatements/
default rewriteBatchedStatements=false
```
INSERT INTO table_name (column1, column2) VALUES (value11, value12);
INSERT INTO table_name (column1, column2) VALUES (value21, value21);
INSERT INTO table_name (column1, column2) VALUES (value32, value32);
```
 rewriteBatchedStatements=true
```
INSERT INTO table_name (column1, column2) VALUES (value11, value12) , (value21, value21),(value32, value32);
```
PTAL: @JNSimba

Reply:
that when using the rewriteBatchedStatements parameter, because the client combines multiple SQL statements into a single statement,

Reply:
But this is only useful for jdbc insert. The writing in sink is done through streamload.

Reply:
> But this is only useful for jdbc insert. The writing in sink is done through streamload.

Sorry, flink sink doris, you did not use jdbc protocol related writing, right?
@wunan1210

Reply:
Yes, use streamload to write

Reply:
Thank you. Then I understand. I will close pr